pkg/app/gcp/instancemanager.go

```diff
@@ -107,6 +106,12 @@ func (m *InstanceManager) CreateHost(zone string, req *apiv1.CreateHostRequest,
 			},
 		},
 		Labels: labels,
+		// Required to enable nested virtualization
+		// https://cloud.google.com/compute/docs/instances/nested-virtualization/enabling#enabling_nested_virtualization_directly_on_a_new_vm
+		MinCpuPlatform: "Intel Haswell",
```

Comment:
This setting is actually only required for n1-* types instances.
It will not work for n2-* instances even if nested virtualization is supported.

So you actually need to add this only is MachineType startswith "n1-"

We found that when working will cloud workstation team to enable cuttlefish on this infra. The documentation is not very clear about that.